tests/integration/test_properties.py

```diff
@@ -38,31 +36,25 @@ def test_post_property(
     assert response.status_code == 400
 
 
-def test_get_property_by_id(client, auth_headers, test_database):
-    """The get property by name returns a successful response code."""
-    response = client.get("/api/properties/1", headers=auth_headers["admin"])
-    property_info = response.get_json()
-    user_json = UserModel.find_by_id(5).json()
-    assert response.status_code == 200
-    assert property_info["name"] == "test1"
-    assert property_info["address"] == "123 NE FLanders St"
-    assert property_info["num_units"] == 5
-    assert property_info["city"] == "Portland"
-    assert property_info["state"] == "OR"
-    assert property_info["zipcode"] == "97207"
-    assert property_info["propertyManager"] == [user_json]
-    assert property_info["propertyManagerName"] == ["Gray Pouponn"]
-    assert property_info["archived"] == 0
-    assert property_info["tenants"] == [TenantModel.find_by_id(1).json()]
+@pytest.mark.usefixtures("client_class", "empty_test_db")
+class TestPropertyRequestsByID:
+    def setup(self):
+        self.endpoint = "/api/properties"
+
+    def test_get_property_by_id(self, create_lease, valid_header):
+        """The get property by name returns a successful response code."""
+        lease = create_lease()
+        property = lease.property
+        response = self.client.get(f"{self.endpoint}/1", headers=valid_header)
+        property_info = response.get_json()
+        assert property_info == property.json()
```

Comment:
There are a lot of temp variables here that make this hard to parse. 
Can we change this to:
```python
        property =create_lease().property
        response = self.client.get(f"{self.endpoint}/{property.id}", headers=valid_header)
        assert response.get_json() == property.json()
```

Also replace the hardcoded :id as that lowers my confidence in this test.